src/connection.js

```diff
@@ -36,7 +36,8 @@ module.exports = function (classes){
           id = ++self.latestId;
 
           self.callbacks[id] = {
-            resolve, reject
+            'resolve': resolve, 

```

Comment:
the linter was throwing a " use 'esversion: 6'" thingie. I didn't know how to specify the version for the linter. Is this version correct or do you want to keep the es6 version ?